README.md

```diff
@@ -6,7 +6,7 @@ to any cron schedule of your choosing.
 
 To start using the Backstage plugin and see a demo, please [book a demo](https://www.cortex.io/demo)!
 
-## Setup and Integration
+## Setup and Integration (old backend model)
```

Comment:
nit: maybe link to the backstage article somewhere? https://backstage.io/docs/backend-system/building-backends/migrating/

just a small thing saying what "old" and "new" refer to